docs/kubernetes.md

```diff
@@ -31,10 +35,9 @@ Stork looks for the _Kubernetes Service_ with the given name (`my-service` in th
 Instead of using the _Kubernetes Service_ IP directly, and let Kubernetes handle the selection and balancing, Stork inspects the service and retrieves the list of _pods_ providing the service.
 Then, it can select the instance.
 
-
 Supported attributes are the following:
 
 | Attribute              | Mandatory    | Default Value     | Description                                                         |
 |-- -------------------- | ------------ | ----------------- | ------------------------------------------------------------------- |
 | `k8s-host`             | No           | _master url_      | The Kubernetes API host                                             |
-| `k8s-namespace`        | No           |                   | The namespace of the service. Use `all` to discover all namespaces. |
\ No newline at end of file
+| `k8s-namespace`        | Yes          |                   | The namespace of the service. Use `all` to discover all namespaces. |
```

Comment:
right now if not specified, it defaults to the namespace that is in the .kube/config or the mounted volume, right?

Reply:
Yes

Reply:
so adding it in the properties is not required

Reply:
I should have elaborate more. After [this modification](https://github.com/smallrye/smallrye-stork/blob/30df8133581fee56b8b5282dfb89a2424c686379/service-discovery/kubernetes/src/main/java/io/smallrye/stork/servicediscovery/kubernetes/KubernetesServiceDiscovery.java#L52), we need to have it in the properties.

Reply:
still, that does not make it mandatory.

Reply:
@aureamunoz it's not needed to have it in properties when it is configured in the environment, then we get it from https://github.com/smallrye/smallrye-stork/blob/30df8133581fee56b8b5282dfb89a2424c686379/service-discovery/kubernetes/src/main/java/io/smallrye/stork/servicediscovery/kubernetes/KubernetesServiceDiscovery.java#L49